oh-my-bash.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
```

Comment:
I think this commit should just be dropped. It will wash out the GitHub history of different files and also duplicates `oh-my-bash.bash` to `oh-my-bash.sh`, but need to check later whether this commit actually contains something essential.

Reply:
This file turned to be generated by just a mistake in synchronization with the master branch. I dropped the related merging commit.